src/main/java/ru/itis/controllers/ProfileController.java

```diff
@@ -30,27 +28,28 @@ public ProfileController(WishListService wishListService, UserService userServic
     @GetMapping("/profile")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("View user profile page")
-    public Map<String, Object> getProfilePage(@RequestHeader(name = "Authorization") String token) {
+    public ResponseDto getProfilePage(@RequestHeader(name = "Authorization") String token) {
         Optional<User> userCandidate = userService.findUserByToken(token);
-        return ResponseUserDto.from(userCandidate.orElse(User.getDefaultUser()));
+        return userCandidate.map(user -> new ResponseDto(HttpStatus.OK, userService.userToMap(user))).orElseGet(() ->
+                new ResponseDto(HttpStatus.OK, "Invalid token"));
     }
 
     @CrossOrigin
     @PostMapping("/profile")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Create new wish list")
-    public WishList createNewWL(@RequestParam String title, @RequestHeader("Authorization") String token) {
-        return wishListService.addNewWishList(title, token);
+    public ResponseDto createNewWL(@RequestParam String title, @RequestHeader("Authorization") String token) {
+        return new ResponseDto(HttpStatus.OK, wishListService.addNewWishList(title, token));
```

Comment:
Bad Aina! Very bad. Remove this class. Use HTTP statuses for giving info on result success.
```
200 OK

{...}
```
instead of `{ status: OK, data: {...} }`

Reply:
fixed